app/controllers/shipit/merge_status_controller.rb

```diff
@@ -113,9 +114,9 @@ class ReferrerParser
 
       def initialize(referrer)
         if URL_PATTERN =~ referrer.to_s
-          @repo_owner = $1.downcase
-          @repo_name = $2.downcase
-          @pull_request_number = $3.to_i
+          @repo_owner = Regexp.last_match(1).downcase
+          @repo_name = Regexp.last_match(2).downcase
+          @pull_request_number = Regexp.last_match(3).to_i
```

Comment:
If you don't want the global, `if (match_info = URL_PATTERN.match(referrer.to_s)` could do the trick